tests/notificator/test_subscriptions.py

```
@@ -0,0 +1,200 @@
+import pytest
+
+from notificator.notificator_config import LIFECYCLE_SUBSCRIPTION
+from notificator.notificator_config import SubscriptionType
+from notificator.subscriptions import fetch_subscribed_org_ids
+from notificator.subscriptions import get_org_ids
+
+
+def _make_settings(*, dev=False, subscriptions_url="https://example.com/subscriptions"):
+    return SimpleNamespace(
+        subscriptions_url=subscriptions_url,
+        tls_cert_path="/tmp/tls/cert.pem",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/tls/cert.pem"

(S108)